Dom.zavd 1/Salon.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+public class Salon
+{
+    private int carCount;
+    public Salon()
+    {
+        carCount = 0;
+    }
+    public void AddCar(string Brand)
+    {
+        carCount++;
+        Console.WriteLine("Car added successfully: " + carCount );

```

Comment:
Remove using Console from inside this class to Program.cs. According to S principle in SOLID https://en.wikipedia.org/wiki/SOLID it should be responsible just for 1 thing. In your case, your class is responsible for counting cars and displaying messages

Reply:
I need more time to fix this(